ssas/service/public/tokens.go

```diff
@@ -54,7 +44,11 @@ func MintRegistrationToken(oktaID string, groupIDs []string) (*jwt.Token, string
 		return nil, "", err
 	}
 
-	return server.MintTokenWithDuration(&claims, selfRegistrationTokenDuration)
+	return server.MintTokenWithDuration(&claims, cfg.SelfRegistrationTokenDuration)
```

Comment:
Do we think we should have a default duration in case the duration isn't set in the config? It looks like it was 60 minutes before.

Reply:
Additionally, if we think the efficiency of checking the env var value each time is a concern, we can also maybe store the default duration in the state of the `TokenCreator` and just read the env var once upon creation

Reply:
Oops, nevermind -- I misunderstood how cfg works in SSAS. Please disregard!